.appveyor.yml

```diff
@@ -23,19 +26,24 @@ test_script:
   # Set environment variables
   - set PYTHONPATH=%APPVEYOR_BUILD_FOLDER%
   - set PATH="%APPVEYOR_BUILD_FOLDER%;C:\Program Files\Wireshark\;%PATH%"
+
+  # Disable open_ssl client tests
+  - set ARGS=-K open_ssl_client
```

Comment:
I think that it is not needed as you are using a Python3 specific tag which is more explicit.

Reply:
See my comment below